src/evaluation/retriever_reader/retriever_reader_eval_squad.py

```diff
@@ -509,7 +509,7 @@ def tune_pipeline(
         parameters,
         parameter_tuning_options,
         elasticsearch_hostname = os.getenv("ELASTICSEARCH_HOSTNAME") or "localhost",
-        elasticsearch_port = int(os.getenv("ELASTICSEARCH_PORT")) or 9200)
+        elasticsearch_port = int(os.getenv("ELASTICSEARCH_PORT") or 9200))
```

Comment:
there is still an extra parentesis no ?

Reply:
No, the missing opening parenthesis is a few lines above at the start of the parameter list.